kokobot/cogs/util.py

```diff
@@ -83,3 +84,20 @@ def __str__(self):
             sent = await ctx.send('Cannot shutdown as {}'.format(ctx.message.author))
             await ctx.message.delete(delay=5)
             await sent.delete(delay=5)
+        
+    @commands.command()
+    async def ava(self, ctx):
+        mentions = ctx.message.mentions
+        
+        if len(mentions) == 0:
+            member = ctx.message.author
+        else:
+            member = mentions.pop(0)
+
+        name = str(member)
+        avatar_url = str(member.avatar_url)
+
+        embed = discord.Embed(colour=2818026)
```

Comment:
Can you add a comment here describing what the color is?

Like: `discord.Embed(...) # blue`

Reply:
Nice, this is fast and looking good.